config/helm/aws-node-termination-handler/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: aws-node-termination-handler
 description: A Helm chart for the AWS Node Termination Handler.
 type: application
-version: 0.18.0
+version: 0.18.1
```

Comment:
Can you remove this update? We increment version when we release (see file's history for more context).

Reply:
done